### PR TITLE
[46] Re-schedule all alarms after rebooting the app 

### PR DESCRIPTION
**Pull request by steve1rm**
# Pull Request template

###  What does this PR solve?

- After rebooting the app all alarms will be scheduled

### CODE REVIEW

File: data/src/main/java/me/androidbox/data/alarm_manager/AlarmBootReceiver.kt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                                agendaId = agendaItem.id,
+                                title = agendaItem.title,
+                                description = agendaItem.description,
+                                remindAt = agendaItem.remindAt,
+                                agendaType = agendaItem.agendaType
+                            )
+                        )
+                    }
+                    
+                    coroutineContext.cancelChildren()
```
Comment (steve1rm) [inline data/src/main/java/me/androidbox/data/alarm_manager/AlarmBootReceiver.kt]: I don't want to keep this coroutines running as the boardcast has finished. So is it good to cancel like this?

Comment (philipplackner) [reply data/src/main/java/me/androidbox/data/alarm_manager/AlarmBootReceiver.kt]: When this job finishes, nothing is running, so this isn't needed

File: data/src/main/java/me/androidbox/data/local/agenda/AgendaLocalRepositoryImp.kt [hunk 2]
```diff
@@ -73,6 +76,30 @@ class AgendaLocalRepositoryImp @Inject constructor(
         eventDao.deleteSyncEventsBySyncType(syncAgendaType)
     }
 
+    override suspend fun fetchAllRemindAtFromEvents(): List<Event> {
+        val eventEntity = eventDao.getAllRemindAt(System.currentTimeMillis())
```
Comment (philipplackner) [inline data/src/main/java/me/androidbox/data/local/agenda/AgendaLocalRepositoryImp.kt:80]: Should be plural right? Also I find `getAllRemindAt` a bit confusing together with `eventEntity`. The function name sounds like it returns remind at times, but apparently it returns event entities? 🤔

**Review by philipplackner — APPROVED** [review]

File: data/src/main/java/me/androidbox/data/alarm_manager/AlarmBootReceiver.kt (hunk 1 above)
Comment (steve1rm) [reply data/src/main/java/me/androidbox/data/alarm_manager/AlarmBootReceiver.kt]: @philipplackner Updated to remove the cancelChildren

**Pull request merged**